CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
```

Comment:
Please:

1. don't use Makdown sub-sections (applies to all changes in this PR, not this line only)
2. group changes by their type (e.g. as it was done in 1.2.0 version: bug fixes, features, misc, test suite, etc.)

---

1. none of current version uses Markdown sub-sections
2. mixing features & bug fixes in a single list doesn't help

Reply:
adjusted